pkg/util/util.go

```diff
@@ -265,33 +264,12 @@ func CheckBackupType(backupTarget string) (string, error) {
 }
 
 func GetBackupCredential(backupURL string) (map[string]string, error) {
```

Comment:
Suggest moving `GetBackupCredential` to util/credential.go as well.

Reply:
Addressed. Thank you.